ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -449,10 +416,27 @@ private static final CloudStorageException toCloudStorageException(String messag
       retryDelayMs = ((DocumentClientException) e).getRetryAfterInMilliseconds();
     }
     // Everything is retryable except NOT_FOUND
-    isRetryable = (statusCode != HttpConstants.StatusCodes.NOTFOUND);
+    boolean isRetryable = (statusCode != HttpConstants.StatusCodes.NOTFOUND);
     return new CloudStorageException(message, e, isRetryable, retryDelayMs);
   }
 
+  /**
+   * Return corresponding {@link AzureReplicationFeed} object for specified {@link AzureReplicationFeedType}.
+   * @param azureReplicationFeedType replication feed type.
+   * @return {@link AzureReplicationFeed} object.
+   */
+  private AzureReplicationFeed getReplicationFeedObj(AzureReplicationFeedType azureReplicationFeedType) {
```

Comment:
Minor: I would pass cosmosDataAccessor, azureMetrics here explicitly rather than relying on constructor to initialize them beforehand.